src/plugins/intel_cpu/src/emitters/snippets/aarch64/cpu_generator.cpp

```diff
@@ -73,6 +78,38 @@ namespace ov {
             }                                                                                        \
     }
 
+#define CREATE_ROUND_V5_EMITTER(e_type_from_zero, e_type_even)                                       \
+    {                                                                                                \
+        [this](const snippets::lowered::ExpressionPtr& expr) -> std::shared_ptr<snippets::Emitter> { \
+            const auto& n = expr->get_node();                                                        \
+            const auto& round = std::dynamic_pointer_cast<ov::op::v5::Round>(n);                     \
```

Comment:
I laucnhed CI and there is one (for now) red GHA job - code style.

When you fix code style, please replace `std::dynamic_pointer_cast` with `ov::as_type_ptr`.
OpenVINO has own RTTI and `ov::as_type_ptr<Type>` is used for casting some variables to special OpenVINO type `Type` if possible.